src/controllers/v2/scheduled.js

```diff
@@ -169,22 +95,29 @@ const ScheduledController = {
       include: [{model: Return}]
     });
   },
+  async findAllDueToExpireInTwoWeeks() {
+    const todaysDate = new Date();
+    const fourteenDaysFromNowStart = addDaysSetTime(todaysDate, 14, 0, 0, 0);
+    const fourteenDaysFromNowEnd = addDaysSetTime(todaysDate, 14, 23, 59, 59);
 
+    return Registration.findAll({
+      include: [
+        {model: Return, required: false},
+        {model: Renewal, required: false}
+      ],
+      where: {
+        '$Renewals.id$': {[Op.is]: null},
```

Comment:
Not sure this needs checked? If a registration has been renewed the expiry date will have been updated anyway? And if a registration is renewed will this not fail the following year?

Reply:
Good shout Paul, ill make this change to #2434 as well as this uses a similar logic

Reply:
Resolved in https://github.com/Scottish-Natural-Heritage/trap-registration-api/pull/134/commits/cee901da338ac43bdfed34ffc4ebdfb4996431e7.